persist/sqlite/wallet.go

```diff
@@ -175,8 +175,13 @@ func (s *Store) UpdateWallet(w wallet.Wallet) (wallet.Wallet, error) {
 // addresses that were previously associated with the wallet.
 func (s *Store) DeleteWallet(id wallet.ID) error {
 	return s.transaction(func(tx *txn) error {
+		_, err := tx.Exec(`DELETE FROM wallet_addresses WHERE wallet_id=$1`, id)
```

Comment:
Is there ever a reason for us to be able to delete a wallet without deleting the wallet addresses? If not a CASCADE would probably also do the trick.